### PR TITLE
Feature/tree size limit

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Limit TREE_SIZE_LIMIT (1,000,000)
- Add testCase

Resolve: #311

### CODE REVIEW

File: unittest/db.test.js [hunk 1]
```diff
@@ -3264,5 +3264,26 @@ describe("Transaction execution", () => {
       const overHeightTx = Transaction.fromTxBody(overHeightTxBody, node.account.private_key);
       assert.deepEqual(node.db.executeTransaction(overHeightTx).code, 23);
     })
+
+    it("blocks over-size transaction", () => {
```
Comment (platfowner) [inline unittest/db.test.js:3268]: minor: s/blocks/rejects/?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: unittest/db.test.js (hunk 1 above)
Comment (cshcomcom) [reply unittest/db.test.js]: Looks better. Done!

**Comment by cshcomcom** [comment]
Thanks for the quick review!

**Pull request merged**